### PR TITLE
add clearfixes to allow using block alignment for the buttons

### LINKED ISSUE #26: Using left or right alignment for buttons causes layout issues
### Reproduction Steps

+ Align either of the buttons to the right or left

### Expected Behavior

+ No layout issues

### Actual Behavior

+ Layout has issues

### Additional Information

This is happening because the alignment "floats" the button to the left or right. This isn't really an issue but the messages (and entire block wrapper) should be cleared so that the alignment doesn't break the layout the way it currently does.

### PR DESCRIPTION
**Pull request by thomasplevy**
## Description

Fixes #26 

## How has this been tested?

+ Manually
+ Existing e2e tests

## Types of changes

+ Uses block deprecation to preserve the previous version of the block
+ Adds a wrapper element and new CSS to ensure buttons can be aligned left or right (using css floats) without breaking the layout

## Checklist:
- [x] My code has been tested.
- [x] My code passes all existing automated tests. 
- [x] My code follows the LifterLMS Coding & Documentation Standards.

### CODE REVIEW

**Comment by thomasplevy** [comment]
@eri-trabiccolo BIG OLD SIGH. One day old and have to add a block deprecation already!

File: src/js/editor/save.js [hunk 1]
```diff
@@ -27,26 +29,47 @@ function getBlockClassName( attributes ) {
...
+	} );
+
+	return innerBlocks;
+}
+
 /**
  * Save function for the block
  *
  * @since 0.0.1
+ * @since [version] Moved `ensureInnerBlockClassNames()` to it's own function.
```
Comment (eri-trabiccolo) [inline src/js/editor/save.js:55]:
```suggestion
 * @since [version] Moved `ensureInnerBlockClassNames()` to its own function.
```

**Review by eri-trabiccolo — CHANGES_REQUESTED** [review]
Minor change.
Although... We're introducing this new clearfix wrapper to always "contain" inner floating elements, right? 
If this is the only reason (as far as I understood it), we could even avoid that and use a nice trick:
```css
.wp-block-llms-lite-lms-progress-tracker {
	clear: both;
	overflow: hidden;
}
```
or
```css
.wp-block-llms-lite-lms-progress-tracker {
	clear: both;
	display: flex;
	flex-direction: column;
}
```

I'm going to push a PR with the first proposal.
see #29

**Comment by thomasplevy** [comment]
Closing in favor of @eri-trabiccolo's solution in #29

**Pull request closed without merging**